Blockzilla/URLBar.swift

```diff
@@ -730,9 +730,8 @@ class URLBar: UIView {
 
 extension URLBar: AutocompleteTextFieldDelegate {
     func autocompleteTextFieldShouldBeginEditing(_ autocompleteTextField: AutocompleteTextField) -> Bool {
-        
-        setTextToURL()
 
+        setTextToURL(displayFullUrl: true)
         autocompleteTextField.highlightAll()
```

Comment:
After playing around with this a bit, I noticed the behavior we have now is moving the cursor to the _end_ of the url (which is not what Safari or Firefox iOS do). 

In order to fix this, we should move this line to our definition of URLTextField, something like

```
func textFieldDidBeginEditing(_ textField: UITextField) {
        guard let autocompleteTextField = textField as? AutocompleteTextField else { return }
        autocompleteTextField.highlightAll()
}
```